src/SprykerEco/Zed/Unzer/Business/UnzerFacadeInterface.php

```diff
@@ -0,0 +1,232 @@
+     * @return bool
+     */
+    public function checkIsAuthorizeFailedOmsCondition(int $idSalesOrderItem): bool;
+
+    /**
+     * Specification:
+     *  - Checks if Unzer Authorization is canceled.
+     *
+     * @api
+     *
+     * @param int $idSalesOrderItem
+     *
+     * @return bool
+     */
+    public function checkIsAuthorizeCanceledOmsCondition(int $idSalesOrderItem): bool;
+
+    /**
+     * Specification:
+     *  - Checks if Unzer Payment is completed.
+     *
+     * @api
+     *
+     * @param int $idSalesOrderItem
+     *
+     * @return bool
+     */
+    public function checkIsPaymentCompletedOmsCondition(int $idSalesOrderItem): bool;
+
+    /**
+     * Specification:
+     *  - Checks if Unzer Charge failed.
+     *
+     * @api
+     *
+     * @param int $idSalesOrderItem
+     *
+     * @return bool
+     */
+    public function checkIsChargeFailedOmsCondition(int $idSalesOrderItem): bool;
+
+    /**
+     * Specification:
+     *  - Checks if Unzer Payment is charged-back.
+     *
+     * @api
+     *
+     * @param int $idSalesOrderItem
+     *
+     * @return bool
+     */
+    public function checkIsPaymentChargebackOmsCondition(int $idSalesOrderItem): bool;
+
+    /**
+     * Specification:
+     *  - Executes Unzer API Charge request.
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     * @param int[] $salesOrderItemIds
+     *
+     * @return void
+     */
+    public function executeChargeOmsCommand(OrderTransfer $orderTransfer, array $salesOrderItemIds): void;
+
+    /**
+     * Specification:
+     *  - Executes Unzer API Refund request.
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\RefundTransfer $refundTransfer
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     * @param int[] $salesOrderItemIds
+     *
+     * @return void
+     */
+    public function executeRefundOmsCommand(
+        RefundTransfer $refundTransfer,
+        OrderTransfer $orderTransfer,
+        array $salesOrderItemIds
+    ): void;
+
+    /**
+     * Specification:
+     *  - Filters available payment methods.
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\PaymentMethodsTransfer $paymentMethodsTransfer
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\PaymentMethodsTransfer
+     */
+    public function filterMarketplacePaymentMethods(
+        PaymentMethodsTransfer $paymentMethodsTransfer,
+        QuoteTransfer $quoteTransfer
+    ): PaymentMethodsTransfer;
+
+    /**
+     * Specification:
+     *  - Get unzer merchant information by merchant reference.
+     *
+     * @api
+     *
+     * @param string $merchantReference
+     *
+     * @return \Generated\Shared\Transfer\MerchantUnzerParticipantTransfer
+     */
+    public function getUnzerMerchantByMerchantReference(string $merchantReference): MerchantUnzerParticipantTransfer;
```

Comment:
Pay attention that we have new RFC [Common entity fetching facade function names](https://spryker.atlassian.net/wiki/spaces/RFC/pages/2594832674/ACCEPTED+Common+entity+fetching+facade+function+names) and should follow.